src/tui/proc-table.go

```diff
@@ -46,13 +47,24 @@ func (pv *pcView) fillTableData() {
 			pv.procTable.RemoveRow(row)
 			continue
 		}
+
+		if pv.processRegex != nil && !pv.processRegex.MatchString(state.Name) {
+			pv.procTable.RemoveRow(row)
+			continue
+		}
+
 		rowVals := getTableRowValues(state)
 		setRowValues(pv.procTable, row, rowVals)
 		if state.IsRunning {
 			runningProcCount += 1
 		}
 		row += 1
 	}
+	if row == 1 {
+		pv.procTable.SetSelectable(false, false)
+	} else {
+		pv.procTable.SetSelectable(true, false)
+	}
```

Comment:
I think you encountered my bug, which I already fixed in my env.
Instead of this block, can you try:
```golang
if pv.procTable.GetRowCount()-1 > row - 1{
```
Notice the `row - 1`
In the line below.

Reply:
I added this block as a workaround for the issue in `tview`. It seems to break if the currently selected row is removed from the table. Which i was also able to replicate in isolation. 

At one point i tried to fix it by calling `table.clear()` before rendering the rows, which means it didn't have to clean up the remaining rows. And it still had the same issue (with the lines you mentioned removed). 

I tried it with this change and run into the same issue.  (to test it)
- enter a filter that results in no processes
- press j or k

Reply:
That's strange. I couldn't replicate it in my environment by, for example, hiding a selected namespace `Ctrl+G`.
Even if it's the last row, leaving me with 0 rows (except the title row).

Reply:
Interesting. How did you filter all rows using namespaces? Doesn't every namespace in the list have at least one project?

Reply:
Weird. If i
- use the process filter to filter everything but nginx. 
- filter away nginx using namespace filter
- press j
i run into the same issue (notice it only crashes when i actually click j)

Reply:
You are right...
Once I click `j` the same happens to me.

Reply:
It doesn't crash though, for me the process compose is stuck and consumes a lot of CPU.

Reply:
Yes sorry, that's what i ment by "crashing" , actually the background processes also keep running fine. (As seen by the logs updating by looking at the logfile directly)

Reply:
I attached a goroutine profiler. It looks like the table is stuck in this loop:
https://github.com/rivo/tview/blob/861aa94d61c899b32a1304d61b840d0178ece408/table.go#L1344

```
#	0x65ae14	github.com/rivo/tview.(*Table).InputHandler.func1.2+0x54			/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/table.go:1375
#	0x65aca3	github.com/rivo/tview.(*Table).InputHandler.func1.5+0xa3			/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/table.go:1440
#	0x65a5a4	github.com/rivo/tview.(*Table).InputHandler.func1+0x4e4				/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/table.go:1598
#	0x65a08f	github.com/rivo/tview.(*Table).InputHandler.(*Box).WrapInputHandler.func2+0x4f	/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167
#	0x63bb16	github.com/rivo/tview.(*Grid).InputHandler.func1+0x1b6				/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/grid.go:277
#	0x63b92f	github.com/rivo/tview.(*Grid).InputHandler.(*Box).WrapInputHandler.func2+0x4f	/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167
#	0x650801	github.com/rivo/tview.(*Pages).InputHandler.func1+0xa1				/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/pages.go:311
#	0x65072f	github.com/rivo/tview.(*Pages).InputHandler.(*Box).WrapInputHandler.func2+0x4f	/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167
#	0x627737	github.com/rivo/tview.(*Application).Run+0x557					/home/eugene/go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/application.go:344
#	0xb3a3fd	github.com/f1bonacc1/process-compose/src/tui.SetupTui+0x13d			/home/eugene/projects/go/process-compose/src/tui/view.go:474
#	0xb7ebb8	github.com/f1bonacc1/process-compose/src/cmd.startTui+0x178			/home/eugene/projects/go/process-compose/src/cmd/project_runner.go:92
```

Reply:
Given this remaining issue, is there anything else in this PR that still needs to be addressed? 
Do you think this workaround should be used for now? GIven that #133 will have a similar issue? Or wait for an upstream fix?

Reply:
I already merged your PR locally.
I believe I will release a new version over the weekend.
Does this timeline work for you?

Reply:
Awesome! no rush, i was just wondering if there were any more changes needed 👌